src/commands/junit/utils.ts

```diff
@@ -1,7 +1,41 @@
+import {SpanTags} from '../../helpers/interfaces'
+import {CI_JOB_URL, CI_PIPELINE_URL} from '../../helpers/tags'
+
 export const getBaseIntakeUrl = () => {
   if (process.env.DATADOG_SITE || process.env.DD_SITE) {
     return `https://cireport-intake.${process.env.DATADOG_SITE || process.env.DD_SITE}`
   }
 
   return 'https://cireport-intake.datadoghq.com'
 }
+
+export const getBaseUrl = () => {
+  if (process.env.DATADOG_SITE || process.env.DD_SITE) {
+    let site = process.env.DATADOG_SITE || process.env.DD_SITE
+    if (site === 'datadoghq.com') {
+      site = 'app.datadoghq.com'
+    } else if (site === 'datad0g.com') {
+      site = 'dd.datad0g.com'
+    }
+
+    return `https://${site}`
+  }
+
+  return 'https://app.datadoghq.com'
+}
+
+export const getTestRunsUrl = (spanTags: SpanTags): string => {
+  if (!spanTags[CI_PIPELINE_URL] && !spanTags[CI_JOB_URL]) {
+    return ''
+  }
+
+  let query = '@test.status:fail'
```

Comment:
why fixed to failed tests?

Reply:
Based on my experience, the majority of the time the user wants to know which tests failed. But again, we can remove it if we want and show every test.

Reply:
yeah the problem, I'd say, is that we'd redirect them to an empty list most of the times